Pipelines/test-execution-pipeline.yml

```diff
@@ -0,0 +1,102 @@
+parameters:
+  - name: IsMultiRegionEnabled
+    type: boolean
+    default: false
+
+  - name: IsClusterRequired
+    type: boolean
+    default: false
+
+  - name: JmeterFolderPath
+    type: string
+    default: JMeterFiles
+
+  - name: JmeterFileName
+    type: string
+    default: SampleTestPlan.jmx
+
+  - name: Threads
+    type: number
+    default: 10
+
+  - name: Duration
+    type: number
+    default: 300
+
+  - name: Loops
+    type: number
+    default: 3
+
+  - name: RampUpTime
+    type: number
+    default: 10
+
+variables:
+  # Tenant: ""
```

Comment:
I forgot why did we keep commented varaiable here

Reply:
Its added for reference so users have two ways of adding values for variables - 
1. they can directly edit the yml file and uncomment variables and pass values
2. refer variables names from here and pass during the run